--- origsrc/libgda-4.2.2/providers/sqlcipher/Makefile.am	2010-11-25 14:59:53.000000000 -0600
+++ src/libgda-4.2.2/providers/sqlcipher/Makefile.am	2010-12-19 20:39:49.000000000 -0600
@@ -21,9 +21,11 @@ libgda_sqlcipher_la_SOURCES = \
 	$(sqlite_sources) \
 	libmain.c
 
-libgda_sqlcipher_la_LDFLAGS = -export-dynamic -module -avoid-version $(LIBTOOL_PROV_EXPORT_OPTIONS)
+libgda_sqlcipher_la_LDFLAGS = -export-dynamic -module -avoid-version $(NO_UNDEFINED) $(LIBTOOL_PROV_EXPORT_OPTIONS)
 libgda_sqlcipher_la_LIBADD = \
 	$(top_builddir)/libgda/sqlite/libgda-sqlcipher.la \
+	$(top_builddir)/libgda/libgda-4.0.la \
+	$(LIBGDA_LIBS) \
 	$(LIBCRYPTO_LIBS)
 
 xmldir   = $(datadir)/libgda-4.0
